web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -96,9 +102,10 @@ const InternetRadioSearch = ({ onChange }) => {
     const [host, setHost] = React.useState("");
     const [results, setResults] = React.useState([]);
     const [query, setQuery] = React.useState("");
+    const [selectedUuid, setSelectedUuid] = React.useState("");
 
     const search = (name) => {
-        setResults([<CircularProgress key={name} />]);
+        setResults([{name: "Loading..."}]);
```

Comment:
Because keeping the state for which item was selected requires dynamically rendered items from the actual data structure itself, the spinner was removed. I've tried a couple of ways to get the `Avatar` component to do anything a little more sophisticated here and a couple of different ways of rendering, but I quickly approached a point of diminished returns on my time and landed on this instead. Let me know if it's a killer feature and I can re-introduce it with fresh eyes.